introduction_to_amazon_algorithms/jumpStart_tabular_regression/Amazon_JumpStart_Tabular_Regression_LightGBM_CatBoost.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
<p>Should the array <code>(num_examples, )</code> have a second item after the comma? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
No, that is the array shape returned from the model.